tests/ios/app/App/main.m

```diff
@@ -2,5 +2,6 @@
 @import OnlySources;
 
 int main(int argc, char **argv) {
+    [[FW alloc] foo];
```

Comment:
Minor - in idiomatic objc objects are expected be initialized prior to calling a method on them - instead you can call `+new` or `[[FW alloc] init]`

Reply:
👍